src/components/pages/FavoriteFoods.tsx

```diff
@@ -23,18 +27,22 @@ export default function FavoriteFoods() {
   } = useFavorites({ username: profileUsername })
   const { error: errorUpdateFavorites, mutate: updateFavorites } = useUpdateFavorites()
   const [favsList, setFavsList] = useState<FavoriteFood[]>(favorites)
+  const [mouseIdx, setMouseIdx] = useState('none')
 
-  const handleSelection = () => {
-    // TODO styling view review button based on mouse hover or click
-    console.log('review pop up')
-  }
+  const Item = styled(Paper)(({ theme }) => ({
```

Comment:
const FoodItem could be better meaningful name.

Reply:
made the change. thanks